plugins/TelescopeControl/src/common/Socket.cpp

```diff
@@ -40,7 +40,11 @@ long long int GetNow(void)
 		qint64 t;
 	} tmp;
 	GetSystemTimeAsFileTime(&tmp.file_time);
-	t = (tmp.t/10) - 86400000000LL*(369*365+89); // TODO: Explain this magic please!
+	// convert from NT/NTFS time epoch (1601-01-01 0:00:00 UTC) in μs to UNIX epoch (1970-01-01 0:00:00 UTC) in μs. The difference is:
+	// 24 hour/day × 3600 s/hour × ((369×365+89) day).
```

Comment:
This misses the 10<sup>6</sup>μs/s factor that I edited into the comment a few minutes after posting it. Without it you get a million times smaller number.